tests/test_document.py

```diff
@@ -47,6 +48,15 @@ def valid_doc_paths(self) -> list[str]:
     def invalid_doc_paths(self) -> list[str]:
         return [get_video_files()[0], get_audio_files()[0], get_image_files()[0]]
 
+    def office_doc_paths(self) -> list[str]:
```

Comment:
Next question :) Why is this special-cased rather than just using the existing `utils.get_documents()` (which already picks up everything in tests/documents) and incorporating into the existing document tests? (Also see the below comment)